compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/metadata/current_fn.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
Did we want to consider naming this directory `context`/`codegen_context`/'gotoctx' instead of `metadata`?